clientV2/components/modals/Modal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ModalWrapper = props => {
+  const { heading, children, openModal, closeModal } = props;
+  let toggle;
+
+  if (!openModal) {
+    toggle = 'none';
+  } else {
+    toggle = 'flex';
+  }
+  return (
+    <div className="modal" style={{ display: toggle }}>
```

Comment:
I think it's not a good practice to use style directly this way, because of it's over-riding behaviour. I think a better approach is to create class with display property of  none and another with display property of flex in your _modal.scss. Then you can do something like 

`<div className={`modal ${openModal ? "flex" : "none"}`} />
`
Following this approach, you wont need the extra if block